WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1773,7 +1778,7 @@ CheckboxTableRightText      = text-align:left; margin-left:@CheckBoxTableCellSpa
 MailAttachmentsTable        = @FullWidth@
 MailAttachmentItem          = @FullWidth@ @AppBg@ margin-bottom:2px; @roundCorners(3px)@ overflow:hidden;
 MailAttachmentItemActive    = background-color: @darken(AppC,13)@;
-MailAttachmentIcon          = @BigIconSize@
+MailAttachmentIcon          = @BigIconSize@; @roundCorners(0px)@ !important;
```

Comment:
Agreed with @tarangkhandelwal as we need to override default rule for this. Also this rule doesn't have any side-effects, as it targets only single element.